### ISSUE TITLE
Carthage build success but got a "Failed to locate resources.bundle" exception

### ISSUE THREAD

**Issue by grimabe** [layerhq/Atlas-iOS#1184]
I am using the latest commit with Carthage fa878c29a78fcaab25e50dad00f8ae56eb8cec3d and this time it successfully built Atlas and Layer (thanks for this fix)

Layer framework seems to be working fine (the auth works) but when I try to use Atlas it crashes :

```
2016-05-08 13:57:51.766 XXX[80272:1709510] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Failed to locate `resources.bundle` in (null)'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000105e82d85 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x00000001083b4deb objc_exception_throw + 48
	2   CoreFoundation                      0x0000000105e82cbd +[NSException raise:format:] + 205
	3   Atlas                               0x000000010720f910 ATLResourcesBundle + 454
	4   Atlas                               0x000000010720ffee -[ATLConversationTableViewCell lyr_commonInit] + 1059
	5   Atlas                               0x000000010720fb77 -[ATLConversationTableViewCell initWithStyle:reuseIdentifier:] + 64
	6   UIKit                               0x00000001089a2a33 -[UITableView _dequeueReusableViewOfType:withIdentifier:] + 640
	7   UIKit                               0x00000001089a2e32 -[UITableView dequeueReusableCellWithIdentifier:forIndexPath:] + 71
	8   Atlas                               0x00000001071f21ff -[ATLConversationListViewController tableView:cellForRowAtIndexPath:] + 104
	9   UIKit                               0x00000001089b64f4 -[UITableView _createPreparedCellForGlobalRow:withIndexPath:willDisplay:] + 766
	10  UIKit                               0x00000001089b662c -[UITableView _createPreparedCellForGlobalRow:willDisplay:] + 74
	11  UIKit                               0x000000010898ad4f -[UITableView _updateVisibleCellsNow:isRecursive:] + 2996
	12  UIKit                               0x00000001089bf686 -[UITableView _performWithCachedTraitCollection:] + 92
	13  UIKit                               0x00000001089a6344 -[UITableView layoutSubviews] + 224
	14  UIKit                               0x0000000108913980 -[UIView(CALayerDelegate) layoutSublayersOfLayer:] + 703
	15  QuartzCore                          0x0000000107e1ac00 -[CALayer layoutSublayers] + 146
	16  QuartzCore                          0x0000000107e0f08e _ZN2CA5Layer16layout_if_neededEPNS_11TransactionE + 366
	17  QuartzCore                          0x0000000107e0ef0c _ZN2CA5Layer28layout_and_display_if_neededEPNS_11TransactionE + 24
	18  QuartzCore                          0x0000000107e033c9 _ZN2CA7Context18commit_transactionEPNS_11TransactionE + 277
	19  QuartzCore                          0x0000000107e31086 _ZN2CA11Transaction6commitEv + 486
	20  UIKit                               0x000000010885372e _UIApplicationHandleEventQueue + 7135
	21  CoreFoundation                      0x0000000105da8301 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	22  CoreFoundation                      0x0000000105d9e22c __CFRunLoopDoSources0 + 556
	23  CoreFoundation                      0x0000000105d9d6e3 __CFRunLoopRun + 867
	24  CoreFoundation                      0x0000000105d9d0f8 CFRunLoopRunSpecific + 488
	25  GraphicsServices                    0x000000010b554ad2 GSEventRunModal + 161
	26  UIKit                               0x0000000108858f09 UIApplicationMain + 171
	27  XXX                              0x000000010561da82 main + 114
	28  libdyld.dylib                       0x000000010a4d792d start + 1
	29  ???                                 0x0000000000000001 0x0 + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by feifanzhou** [comment]
@grimabe Thanks for reporting, I expect to have a fix out by tomorrow

**Comment by feifanzhou** [comment]
Resolved with #1192
